src/latent_gp.jl

```diff
@@ -0,0 +1,29 @@
+"""
+    LatentGP(f<:GP, x<:AbstractVector, Φ)
+
+ - `fx` is a `FiniteGP`.
+ - `Φ` is the log likelihood function which maps input of type `Tx` to `Real`.
+    
+"""
+struct LatentGP{T<:AbstractGPs.FiniteGP}
+    fx::T
+    Φ
```

Comment:
```suggestion
struct LatentGP{T<:AbstractGPs.FiniteGP,S}
    fx::T
    Φ::S
```
Maybe also don't use unicode characters here since it's not just an internal implementation detail but users might want to access the field.